experimental/packages/exporter-logs-otlp-grpc/README.md

```diff
@@ -0,0 +1,96 @@
+# OpenTelemetry Collector Logs Exporter for node with grpc
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This module provides exporter for node to be used with OTLP (`grpc`) compatible receivers.
+Compatible with [opentelemetry-collector][opentelemetry-collector-url] versions `>=0.75.0`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/exporter-logs-otlp-grpc
+```
+
+## Service Name
+
+The OpenTelemetry Collector Exporter does not have a service name configuration.
+In order to set the service name, use the `service.name` resource attribute as prescribed in the [OpenTelemetry Resource Semantic Conventions][semconv-resource-service-name].
+To see documentation and sample code for the traces exporter, as well as instructions for using TLS, visit the [Collector Trace Exporter for web and node][trace-exporter-url].
+To see documentation and sample code for the metric exporter, see the [exporter-metrics-otlp-grpc package][metrics-exporter-url]
+
+## Logs in Node - GRPC
+
+The OTLPLogsExporter in Node expects the URL to only be the hostname. It will not work with `/v1/logs`. All
+options that work with trace also work with logs.
+
+```js
+import {
+  LoggerProvider,
+  SimpleLogRecordProcessor,
+} from '@opentelemetry/sdk-logs';
+import { OTLPLogsExporter } from '@opentelemetry/exporter-logs-otlp-grpc';
+
+const collectorOptions = {
+  // url is optional and can be omitted - default is http://localhost:4317
+  url: 'http://<collector-hostname>:<port>',
+};
+
+const loggerExpoter = new OTLPLogsExporter(collectorOptions);
+const loggerProvider = new LoggerProvider();
+
+loggerProvider.addLogRecordProcessor(
+  new SimpleLogRecordProcessor(loggerExpoter)
+);
+
+['SIGINT', 'SIGTERM'].forEach(signal => {
+  process.on(signal, () => loggerProvider.shutdown().catch(console.error));
+});
+
+// logging
+const logger = loggerProvider.getLogger('example-logger');
+logger.emit({ body: 'example-log' });
+```
+
+## Environment Variable Configuration
+
+| Environment variable | Description |
+|----------------------|-------------|
+| OTEL_EXPORTER_OTLP_LOGS_ENDPOINT | The endpoint to send logs to. By default `localhost:4317` will be used. |
+| OTEL_EXPORTER_OTLP_ENDPOINT | The endpoint to send trace, metric, and logs to. By default `localhost:4317` will be used. |
+| OTEL_EXPORTER_OTLP_LOGS_COMPRESSION | The compression type to use on OTLP logs requests. Options include gzip. By default no compression will be used. |
```

Comment:
I cannot see this used anywhere. Is this env var used in this implementation? :thinking: 

Same for `OTEL_EXPORTER_OTLP_LOGS_INSECURE`,
`OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE`,
`OTEL_EXPORTER_OTLP_LOGS_CLIENT_KEY`,
`OTEL_EXPORTER_OTLP_LOGS_CLIENT_CERTIFICATE`,
`OTEL_EXPORTER_OTLP_LOGS_TIMEOUT` :thinking:

Reply:
I see, there is `OTEL_EXPORTER_OTLP_TRACES_COMPRESSION` used in the base packages and need to be fixed (both for metrics and logs). Can I just remove these env vars for now and fix it in another PR?

Reply:
After some deeper investigation, I found that many code need to be fixed and it would be a big change. I prefer keep these env here for now and fix the problems later.

- OTLP-related metrics exporters don't use their env vars too.
- `otlp-exporter-base` and `otlp-grpc-exporter-base` use trace-related env vars in their code.

Related code lines found by now:
[exporter-metrics-otlp-grpc/README.md](https://github.com/open-telemetry/opentelemetry-js/blob/de354dbbdd24bb9f93edef6d098fe8484c7bc3ff/experimental/packages/opentelemetry-exporter-metrics-otlp-grpc/README.md?plain=1#L62)
[exporter-metrics-otlp-http/README.md](https://github.com/open-telemetry/opentelemetry-js/blob/de354dbbdd24bb9f93edef6d098fe8484c7bc3ff/experimental/packages/opentelemetry-exporter-metrics-otlp-http/README.md?plain=1#L82)
[exporter-metrics-otlp-proto/README.md](https://github.com/open-telemetry/opentelemetry-js/blob/de354dbbdd24bb9f93edef6d098fe8484c7bc3ff/experimental/packages/opentelemetry-exporter-metrics-otlp-proto/README.md?plain=1#L52)
[otlp-exporter-base/src/util.ts](https://github.com/open-telemetry/opentelemetry-js/blob/de354dbbdd24bb9f93edef6d098fe8484c7bc3ff/experimental/packages/otlp-exporter-base/src/util.ts#L96)
[otlp-exporter-base/src/platform/node/util.ts](https://github.com/open-telemetry/opentelemetry-js/blob/de354dbbdd24bb9f93edef6d098fe8484c7bc3ff/experimental/packages/otlp-exporter-base/src/platform/node/util.ts#L222)
[otlp-grpc-exporter-base/src/util.ts](https://github.com/open-telemetry/opentelemetry-js/blob/de354dbbdd24bb9f93edef6d098fe8484c7bc3ff/experimental/packages/otlp-grpc-exporter-base/src/util.ts#L173)

Reply:
I think adding support for it in a follow-up should be fine so that we can keep the diff manageable. :slightly_smiling_face: 
We should, however, remove the unsupported ones from `README.md` for now.

Reply:
I created a follow-up for adding the env vars #3747